### PR TITLE
Improved notebook on exogenous regressors

### PR DESCRIPTION
**Pull request by MMenchero**

### CODE REVIEW

File: nbs/examples/Exogenous.ipynb [hunk 1]
```diff
@@ -7,58 +7,111 @@
    "source": [
```
Comment (FedericoGarza) [inline nbs/examples/Exogenous.ipynb:7]: <p>Could we change the colab link? It directs to the cross validation nb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (FedericoGarza) [inline nbs/examples/Exogenous.ipynb:7]: <p>here could we include a more detailed description of how to include exogenous variables? Something like,</p><p> </p><pre class="ql-syntax" spellcheck="false">statsforecast considers as exogenous variables all columns after the column <code>y</code>
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Review by FedericoGarza — CHANGES_REQUESTED** [review]
nice @MMenchero! the changes look great!🙌I left a couple of comments

File: nbs/examples/Exogenous.ipynb (hunk 1 above)
Comment (MMenchero) [reply nbs/examples/Exogenous.ipynb:7]: <p>Done</p>

Comment (MMenchero) [reply nbs/examples/Exogenous.ipynb:7]: <p>Done. </p>

**Pull request closed without merging**